.travis.yml

```diff
@@ -3,7 +3,10 @@
 # Instead use the files under the "/sample_files" directory for that purpose.
 language: python
 sudo: false
-cache: pip
```

Comment:
To fix cache of pip:
 - Remove `cache:pip` (currently isn't working)
 - Add directory pip
```
  directories:
    - $HOME/.pip-cache/
```